examples/python/evm/get_or_create_smart_account.py

```diff
@@ -0,0 +1,32 @@
+# Usage: uv run python evm/get_or_create_account.py
+
+import asyncio
+
+from cdp import CdpClient
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+async def main():
+    async with CdpClient() as cdp:
+        # Single account creation example
+        name = "Account1"
+        account = await cdp.evm.get_smart_account(name=name)
+        print("Account Address: ", account.address)
+        account2 = await cdp.evm.get_smart_account(name=name)
+        print("Account 2 Address: ", account2.address)
+        print("Are accounts equal? ", account.address == account2.address)
+
+        # Concurrent account creation example
+        account_coros = [
+            cdp.evm.get_or_create_smart_account(name="Account"),
+            cdp.evm.get_or_create_smart_account(name="Account"),
+            cdp.evm.get_or_create_smart_account(name="Account"),
```

Comment:
`owner` is required when creating a smart account, so you will have to pass in the owner for this account!

Reply:
@superadi04  can you run this locally to make sure it works after your changes? thx